style.css

```diff
@@ -593,29 +558,15 @@ ul li {
   font-weight: 100;
   font-size: 20px;
   line-height: 28px;
-  position: absolute;
-  height: 28px;
-  width: 427px;
-  top: 470px;
-  right: 0;
-  left: 510px;
 }
 
 .inputtxt3 {
-  position: relative;
-  left: 24px;
-  right: 0;
-  top: 400px;
-  bottom: 0;
-  height: 170px;
-  width: 327px;
   background: #fff;
   border: 1px solid #cfd8dc;
+  border-radius: 8px;
```

Comment:
- [ ] Kindly consider increasing the padding for your input text as it looks really close to the input border in contrast to what we have in the design, please take a close look at it below and implement the changes and also consider increasing the width of the get in touch  button  as well to make an exact duplicate of the provided design 
- your design 👇 
<img width="351" alt="Screen Shot 2023-05-11 at 13 45 26" src="https://github.com/ntimanu/Portfolio/assets/60146030/01384fd6-3f7b-4a5a-8866-06296bb457dd">

- Figma design 👇 
<img width="362" alt="Screen Shot 2023-05-11 at 13 48 31" src="https://github.com/ntimanu/Portfolio/assets/60146030/77d918e9-eb74-4b17-b666-17aac448e7ef">

Reply:
Done